tracing-subscriber/Cargo.toml

```diff
@@ -29,6 +29,7 @@ fmt = ["registry"]
 ansi = ["fmt", "ansi_term"]
 registry = ["sharded-slab", "thread_local"]
 json = ["tracing-serde", "serde", "serde_json"]
+reload = []
```

Comment:
We actually *can't* backport this change to 0.2, even making it a default feature is still a breaking change — if someone is using `tracing-subscriber` with `default-features = false`, it will currently be enabled for them, and feature flagging it means that it'll disappear. 

Unfortunately, because of that, *any* feature flagging of an API that was previously not feature flagged is a breaking change. So, this can go in the next breaking `tracing-subscriber` release for sure, but it can't be backported.

Reply:
Damn, that's true. Forgot about that.